### ISSUE TITLE
[Anon] Bug report: Astra_Militarum.catz

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/wh40k-mobile-format#126]
**File:** Astra_Militarum.catz

**BattleScribe version:** 2.01.04

**Platform:** Windows

**Dropbox:** No

**Description:** Astra Telepathica units (Primaris Psyker, Wyrdvane Psyker, Astropath) are missing. As they have the Astra Militarum keyword in the Index Imperium 2, they should be accessible to Astra Militarum Rosters.

**Comment by Ponha** [comment]
Thats strange, last i checked they where accessible by including the telepathica auxillaries or what ever the selection was called, ill look in to it though

**Comment by Ponha** [comment]
Yeah you get it by adding adeptus astra telepathica and astra telepahica auxillaries (yes thats a typo in the file) under 'Faction. I guess we should fix that thing.

**Comment by LordDarien** [comment]
[WH40k 8th Mobile.zip](https://github.com/BSData/wh40k-mobile-format/files/1260374/WH40k.8th.Mobile.zip)
I'm the guy who posted the bug report.

I figured you'd have to choose the astra telepathica auxiliaries to get access to those psykers I mentioned. But even if I do so, they Primaris Psyker doesn't show up under the HQ sections, nor do the Astropath and Wyrdvanes in the Elites.

I've attached the data file that I have.
